File: java-sfn-unit-test/src/main/java/com/sfnut/myapp/child/Step10_Lambda.java
package com.sfnut.myapp.child;

import com.sfnut.sfn.lambda.SfnLambda;

public class Step10_Lambda extends SfnLambda<ChildPayload> {
    @Override
    public ChildPayload execute(ChildPayload input) {
        input.setHappyPathData(input.getHappyPathData() + " > Child 10");
        return input;
    }
}

File: myapp/child/Step10_Lambda.cs
using sfn_ut.sfn.lambda;

namespace sfn_ut.myapp.child;

public class Step10_Lambda : SfnLambda<ChildPayload> {
  public override ChildPayload Execute(ChildPayload input) {
    input.HappyPathData += " > Child 10";
    return input;
  }
}
